src/extension/app/aem-sidekick.js

```diff
@@ -26,14 +31,20 @@ export class AEMSidekick extends LitElement {
   constructor(config) {
     super();
     this.theme = 'light';
-    appStore.setSiteConfig(config);
+    appStore.loadContext(this, config);
+
+    this.addEventListener('contextloaded', (data) => {
+      // eslint-disable-next-line no-console
+      console.log('console was loaded', data);
+    });
   }
 
   async connectedCallback() {
     super.connectedCallback();
     this.getTheme();
 
     window.matchMedia('(prefers-color-scheme: dark)').addEventListener('change', () => {
+      // istanbul ignore next
```

Comment:
I tried some things but I was unable to get this event to fire by changing the color scheme after the element has loaded.. Settled for just changing the color scheme before load and testing that..